From b9451b22210e3b94b75b2a4dfebc3a7f2e185246 Mon Sep 17 00:00:00 2001
From: B K Karthik <karthik.bk2000@live.com>
Date: Mon, 22 Jun 2020 13:11:06 -0400
Subject: [PATCH 1/2] add jiffies file

---
 task08/task08.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/task08/task08.c b/task08/task08.c
index c662bbc..79a9836 100644
--- a/task08/task08.c
+++ b/task08/task08.c
@@ -54,6 +54,7 @@ static int __init my_init(void)
 {
 	eudy = debugfs_create_dir("eudyptula", NULL);
 	debugfs_create_file("id", 0666, eudy, NULL, &id_fops);
+	debugfs_create_u32("jiffies", 0444, eudy, (u32*)&jiffies);
 	pr_debug("Hello World!\n");
 	return 0;
 }
-- 
2.20.1

